de.metas.dimension/src/main/java/de/metas/dimension/IDimensionspecDAO.java

```diff
@@ -73,4 +73,13 @@
 	 * @return
 	 */
 	List<String> retrieveAttributeValueForGroup(String dimensionSpectInternalName, String groupName, IContextAware ctxAware);
+
+	/**
+	 * @param internalName not empty or null
+	 * @param ctx
+	 * @param trxName
+	 * 
+	 * @return
+	 */
+	I_DIM_Dimension_Spec retrieveForInternalName(String internalName, Properties ctx, String trxName);
```

Comment:
First I thought that context was the reason of my stackoverflow in the console and I forgot to change it back and test after I found out it wasn't.
Fixed now.